tests/scripts/create_test_suffix.py

```diff
@@ -1,3 +1,8 @@
+"""
+Creates a unique suffix for the index name to avoid conflicts
+with another testing executed in the same account.
```

Comment:
nit, language could be more precise:
`testing executed in the same account.` -> `testing executed in the same Marqo cloud account.`